source/code/plugin/in_kube_events.rb

```diff
@@ -141,7 +139,7 @@ def parse_and_emit_records(events, eventQueryState, newEventQueryState, batchTim
           record["Name"] = items["involvedObject"]["name"]
           record["Reason"] = items["reason"]
           record["Message"] = items["message"]
-          record["Type"] = items["type"]
+          record["KubeEventType"] = items["type"]
```

Comment:
This change was already deployed with the previous release.. was there a conflict for this?